packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerHighlight/utils/common/bboxUtils.ts

```diff
@@ -0,0 +1,55 @@
+import { Bbox, TextSpan } from '../../types';
+import { bboxesIntersect } from '../../../../utils/box';
+import { spanIntersection, spanLen } from '../../../../utils/textSpan';
+
+/**
+ * Get bbox for a text span assuming each character takes horizontal spaces evenly
+ * @param bbox bbox occupied with a text
+ * @param origLength length of the text
+ * @returns bbox for the text
+ */
+export function bboxGetSpanByRatio(bbox: Bbox, origLength: number, span: TextSpan): Bbox {
+  const theSpan = spanIntersection([0, origLength], span);
+  if (origLength === 0 || spanLen(theSpan) <= 0) {
+    return [bbox[0], bbox[1], bbox[0], bbox[3]] as Bbox;
+  }
+
+  const [spanStart, spanEnd] = span;
+  const [left, top, right, bottom] = bbox;
+  const width = right - left;
+  const resultLeft = left + (width / origLength) * spanStart;
+  const resultRight = left + (width / origLength) * spanEnd;
+
+  return [resultLeft, top, resultRight, bottom];
+}
+
+/**
+ * Check whether the two bboxes are next to each other in a row.
+ * This is used to get a text of a line from a list of small text cells.
+ */
+export function isNextToEachOther(boxA: Bbox, boxB: Bbox): boolean {
+  if (bboxesIntersect(boxA, boxB)) {
+    return false;
+  }
+
+  const [leftA, topA, rightA, bottomA] = boxA;
+  const [leftB, topB, rightB, bottomB] = boxB;
+  const heightA = bottomA - topA;
+  const heightB = bottomB - topB;
+
+  // compare height ratio
+  const OVERLAP_RATIO = 0.8;
```

Comment:
1. Move this constant out of the function and to the top of the file.
2. Also, what is the purpose of this constance? How was it calculated? Add a description comment.